torch/nn/modules/container.py

```diff
@@ -890,7 +893,10 @@ def extra_repr(self) -> str:
         for k, p in self.items():
             if isinstance(p, torch.Tensor):
                 size_str = 'x'.join(str(size) for size in p.size())
-                device_str = '' if not p.is_cuda else ' (GPU {})'.format(p.get_device())
+                if p.device.type in ["cuda", torch._C._get_privateuse1_backend_name()]:
+                    device_str = ' ({})'.format(p.device)
```

Comment:
It would be nice to preserve the `GPU 1` printing instead of just leaving `1`.

Reply:
> It would be nice to preserve the `GPU 1` printing instead of just leaving `1`.

yeah, I use `p.device` to get the device info, as `cuda`, the `device_str` will be `'(cuda:0)'`, maybe it is ok?

Reply:
Yes that is ok!